recipes-tag/xpdacq/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   noarch: python
-  number: 1
+  number: 0
```

Comment:
Should the corresponding version be updated as well? Or is it still `0.8.1`?

Reply:
Ah maybe this should be left off, I'm not certain why we had a build number bump on your side, but I'll remove it.